_posts/2017-01-15-webapps.markdown

```diff
@@ -78,15 +78,10 @@ docker port static-site
 80/tcp -> 0.0.0.0:32773
 ```
 
-If you are running [Docker for Mac](https://docs.docker.com/docker-for-mac/), [Docker for Windows](https://docs.docker.com/docker-for-windows/), or Docker on Linux, you can open `http://localhost:[YOUR_PORT_FOR 80/tcp]`. For our example this is `http://localhost:32773`.
 
-If you are using Docker Machine on Mac or Windows, you can find the hostname on the command line using `docker-machine` as follows (assuming you are using the `default` machine).
+If you were running [Docker for Mac](https://docs.docker.com/docker-for-mac/), [Docker for Windows](https://docs.docker.com/docker-for-windows/), or Docker on Linux, you would see your app at `(http://localhost:80)`.
 
-```
-$ docker-machine ip default
-192.168.99.100
-```
-You can now open `http://<YOUR_IPADDRESS>:[YOUR_PORT_FOR 80/tcp]` to see your site live! For our example, this is: `http://192.168.99.100:32773`.
+However, you can see it now [here](#){:data-term=".term1"}{:data-port="32773"}
```

Comment:
The problem with this is the that the port is going to be variable. It won't always be 32773. So we can't actually link to it. We should probably just remove the link all together.